@here/harp-lrucache/lib/LRUCache.ts

```diff
@@ -91,8 +97,9 @@ export class LRUCache<Key, Value> {
     ): void {
         let entry = this.m_newest;
         while (entry !== null) {
+            const older = entry.older;
```

Comment:
what does this do?

Reply:
Key the oldest reference, before it will be changed in the callback itself... We do not have any guaranty that callback will not modify (say delete) currently processed item. Actually it was already done in VisibleTileSet.markDataSourceTilesDirty().

Reply:
Ok, sounds like a valid point. Please add a comment in the code to make it clear for others.
Also, the note above says that the results are undefined if the cache is modified during iteration, is that no longer correct? It seems that the cache can be modified and this will still work.

Reply:
Cache element may be modified, but please note that there is also Map<K, V> reference passed in, so entire cache can not be modified. Thus my update in method comments.